### DISCUSSION TITLE
Graph Enabled Llama Index - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#25] [Comments]
# Graph Enabled Llama Index - siwei.io

How Graph could help build better In-context Learning LLM Applications.

https://siwei.io/en/graph-enabled-llama-index/

**Comment by wey-gu** [comment]
Update:
- [x] https://github.com/jerryjliu/llama_index/pull/2581 , WIP, initial PR to introduce graph store
- [ ] e2e tutorial(in llama index repo)
- [ ] demo project

**Reply by younes-io** [reply]
do we have an E2E tuto ? I need this for a real use case
